### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by jennyEckstein**

### CODE REVIEW

File: .github/workflows/pull-request-check.yml [hunk 1]
```diff
@@ -7,20 +7,21 @@ on:
...
-  test:
-    name: Node ${{ matrix.node_version }} on ${{ matrix.os }}
+  check:
+    name: Node ${{ matrix.node-version }} on ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        node_version: [10, 12, 13]
-        os: [ubuntu-latest, windows-latest, macOS-latest]
+        node-version: [12, 14, 15]
```
Comment (eaviles) [inline .github/workflows/pull-request-check.yml:15]: Add also "10"

File: .github/workflows/pull-request-check.yml [hunk 2]
```diff
@@ -34,4 +35,4 @@ jobs:
       - name: Report coverage
         run: npx codecov
         env:
-          CODECOV_TOKEN: ${{secrets.CODECOV_TOKEN}}
+          CODECOV_TOKEN: ${{secrets.CODECOV_TOKEN}}
```
Comment (eaviles) [inline .github/workflows/pull-request-check.yml:38]: No new line?

**Review by eaviles — CHANGES_REQUESTED** [review]

**Review by BizzlePop34 — APPROVED** [review]

File: .github/workflows/pull-request-check.yml (hunk 1 above)
Comment (jennyEckstein) [reply .github/workflows/pull-request-check.yml]: done

File: .github/workflows/pull-request-check.yml (hunk 2 above)
Comment (jennyEckstein) [reply .github/workflows/pull-request-check.yml]: added

**Review by eaviles — APPROVED** [review]

**Pull request merged**